apps/public-docsite-v9/.storybook/fix-title.js

```diff
@@ -1,6 +1,11 @@
 const fs = require('fs');
 const path = require('path');
 
+/**
+ *
+ * @param {string} filePath
```

Comment:
starting with this PR, docsite-v9 storybook related files are strictly type checked to provide best possible DX ( follows common approach we use for v9 projects )